pkg/test/framework/features/allowlist.txt

```diff
@@ -1510,6 +1510,7 @@ telemetry,TestDashboard/istio-performance-dashboard.json
 telemetry,TestDashboard/istio-service-dashboard.json
 telemetry,TestDashboard/istio-workload-dashboard.json
 telemetry,TestDashboard/pilot-dashboard.json
+telemetry,TestDashboard/istio-extension-dashboard.json
```

Comment:
should we add to features labels instead, this was supposed to be for the tests before

From top of the this file: 
`This file contains all test cases known to exist before 6/8/2020
// It will be used to allow old test cases to lack feature labels,
// while failing on new tests that don't use labels.`